### PR TITLE
adding styling for the submission section and matching wireframes

### PR DESCRIPTION
**Pull request by samhstn**
Styling for the suggestions section matching a bit of [this](https://github.com/LDMW/app/issues/73#issuecomment-299250790) and the [wireframes](http://f86j98.axshare.com/#g=1&p=home)

See #59

### CODE REVIEW

**Review by SimonLab — APPROVED** [review]
LGTM

File: web/templates/homepage/index.html.eex [hunk 1]
```diff
@@ -53,12 +53,12 @@
   <% end %>
 </section>
 
-<section class="lm-bg-grey pa1 ph3 pb4 pb5-l" id="suggestion_form">
+<section class="pa4 tc-ns lm-bg-grey pa1 ph3 pb4 pb5-l" id="suggestion_form">
```
Comment (Danwhy) [inline web/templates/homepage/index.html.eex]: Shouldn't the background for this be white? https://github.com/LDMW/app/issues/73#issuecomment-299250790

File: web/templates/homepage/index.html.eex [hunk 2]
```diff
@@ -53,12 +53,12 @@
...
   <%= raw(@lookingfor) %>
   <%= form_for @conn, homepage_path(@conn, :submit_email) <> "#suggestion_form", [as: :suggestions], fn f -> %>
     <%= label(f, :suggestions, "Suggestions", class: "dn") %>
-    <%= textarea(f, :suggestions, placeholder: "-- add your suggestions for new topics --", class: "h4") %>
-    <%= component "primary_button", value: "Take part", class: "right-2" %>
+    <%= textarea(f, :suggestions, placeholder: "Add your suggestions for new topics", class: "h4 w-100") %>
+    <%= component "primary_button", value: "Submit", class: "bottom--1 right-2 mt2" %>
```
Comment (Danwhy) [inline web/templates/homepage/index.html.eex]: This button is showing on the left, it should be on the right

**Review by Danwhy — CHANGES_REQUESTED** [review]
A couple of small changes to match the design

**Review by Danwhy — APPROVED** [review]

**Pull request merged**